src/components/dashboard/components/dictionary/common/DictionaryModal.jsx

```diff
@@ -82,9 +99,30 @@ export class DictionaryModal extends React.Component {
         repository_type,
       },
       errors: {},
+      supportedLocalesOptions,
     });
   }
 
+ preSelectSupportedLocales = () => {
+   const {
+     dictionary: {
+       supported_locales,
+     },
+   } = this.props;
+   if (supported_locales === undefined) {
+     return null;
+   }
+   const supportedLocalesOptions = [];
+   for (let i = 0; i < supported_locales.length; i += 1) {
```

Comment:
You could have used the short form i++ but that is also okay.

Reply:
Thanks for the feedback. However eslint flags it if I use i++.